### PR TITLE
BZ1975509: Added cpu and memory reservation when Latency Sensitivity is set to High

### PR DESCRIPTION
**Pull request by darshan-nagaraj**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1975509

OCP version: **4.7 and 4.6**

Direct Doc Preview link: https://deploy-preview-41684--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc-user-infra.html#installation-vsphere-machines_installing-vmc-user-infra

Peer-review-squad: back-port this PR to 4.6 as well.

@jinyunma PTAL and let me know your feedback.

### CODE REVIEW

File: modules/installation-vsphere-machines.adoc [hunk 1]
```diff
@@ -156,7 +156,7 @@ $ export IPCFG="ip=192.168.100.101::192.168.100.254:255.255.255.0:::none nameser
 $ govc vm.change -vm "<vm_name>" -e "guestinfo.afterburn.initrd.network-kargs=${IPCFG}"
 ----
 +
-*** Optional: In the event of cluster performance issues, from the *Latency Sensitivity* list, select *High*.
+*** Optional: In the event of cluster performance issues, from the *Latency Sensitivity* list, select *High*. You also need to set the VM's CPU and memory reservation to its configured memory size.
```
Comment (jinyunma) [inline modules/installation-vsphere-machines.adoc:159]: How about change to "You also need to set VM's CPU reservation and memory reservation"?

**Comment by jinyunma** [comment]
I think this change should be applied on 4.6+, not only on 4.6 and 4.7.

File: modules/installation-vsphere-machines.adoc (hunk 1 above)
Comment (darshan-nagaraj) [reply modules/installation-vsphere-machines.adoc:159]: @jinyunma I believe there should be some reference on VM's CPU and memory reservation. There are chances that user might get puzzled on how much the reservation should be.

**Comment by darshan-nagaraj** [comment]
> I think this change should be applied on 4.6+, not only on 4.6 and 4.7.

@jinyunma Yes, you're correct. Sorry my bad. I forgot to mention that I have created another PR for 4.8+  at https://github.com/openshift/openshift-docs/pull/41685

**Comment by jinyunma** [comment]
LGTM

File: modules/installation-vsphere-machines.adoc [hunk 2]
```diff
@@ -156,7 +156,9 @@ $ export IPCFG="ip=192.168.100.101::192.168.100.254:255.255.255.0:::none nameser
 $ govc vm.change -vm "<vm_name>" -e "guestinfo.afterburn.initrd.network-kargs=${IPCFG}"
 ----
 +
-*** Optional: In the event of cluster performance issues, from the *Latency Sensitivity* list, select *High*.
+*** Optional: In the event of cluster performance issues, from the *Latency Sensitivity* list, select *High*. Ensure that your VM's CPU and memory reservation have the following values:
+**** Memory reservation value should be equal to its configured memory size.
+**** CPU reservation value should be at least the number of low latency virtual CPUs multiplied by the measured physical CPU speed.
```
Comment (ahardin-rh) [inline modules/installation-vsphere-machines.adoc:161]: we tend to say "must" instead of "should"

**Pull request merged**